jni/mkxp/src/main.cpp

```diff
@@ -262,7 +262,7 @@ int main(int argc, char *argv[])
 	SDL_GL_SetAttribute(SDL_GL_RED_SIZE, 8);
 	SDL_GL_SetAttribute(SDL_GL_GREEN_SIZE, 8);
 	SDL_GL_SetAttribute(SDL_GL_BLUE_SIZE, 8);
-	SDL_GL_SetAttribute(SDL_GL_DEPTH_SIZE, 24);
+	SDL_GL_SetAttribute(SDL_GL_ALPHA_SIZE, 0);
```

Comment:
Might not be relevant for performance, but I just noticed SDL sets the depth size to 16 by default. You could actually set that to 0 to avoid allocating a depth buffer at all.